lib/experimental/new_embed.dart

```diff
@@ -614,6 +615,109 @@ class FlashBox {
   void hide() {
     _element.setAttr('hidden');
   }
+
+  void show() {
+    _element.clearAttr('hidden');
+  }
+}
+
+class AnalysisResultsElement extends DElement {
```

Comment:
You've inherited from DElement here, and the other components we've got are (as far as I can remember) using composition instead. We should probably pick one pattern and stick to it where possible. Do you have strong feelings on extending DElement here, or would it be nbd to switch?